$OpenBSD: patch-make_toplevel_mak,v 1.1 2014/04/25 11:17:27 ajacoutot Exp $

We only use versioned libraries (.so.X.Y versus .so).
(PTLIB_FILE = PTLIB_SONAME)

--- make/toplevel.mak.orig	Mon Feb 17 03:14:37 2014
+++ make/toplevel.mak	Sat Apr 19 20:07:46 2014
@@ -166,18 +166,6 @@ install:
 		$(INSTALL) -m 755 $$lib $(DESTDIR)$(libdir); \
 	  fi ) \
 	done )
-	( if test -e $(PTLIB_LIBDIR)/$(PTLIB_SONAME); then \
-	    (cd $(DESTDIR)$(libdir) ; \
-		rm -f $(PTLIB_FILE) ; \
-		ln -sf $(PTLIB_SONAME) $(PTLIB_FILE) \
-	    ) \
-	fi )
-	( if test -e $(PTLIB_LIBDIR)/$(PTLIB_DEBUG_SONAME); then \
-	    (cd $(DESTDIR)$(libdir) ; \
-		rm -f $(PTLIB_DEBUG_FILE) ; \
-		ln -sf $(PTLIB_DEBUG_SONAME) $(PTLIB_DEBUG_FILE) \
-	    ) \
-	fi )
 ifeq (1, $(HAS_PLUGINS))
 	if test -e $(PTLIB_LIBDIR)/device/; then \
 	cd $(PTLIB_LIBDIR)/device/; \
